### PR TITLE
feat: added video to show up on the NFT detail screen

### PR DESCRIPTION
**Pull request by roshaans**
Users are unable to see their video NFTs on the NFT-detail screen when they click their NFTs. Video NFTs are not supported. 

### What Changed? 
- Users are able to video and regular image NFTs in the NFT details tab. 

| Before   |      After      |
|----------|:-------------:|
| Before | After |

## Image NFTs show up as normal 

| Collectible Screen   |      NFT Detail Screen      |
|----------|:-------------:|
| image | image|

### CODE REVIEW

**Comment by andy-haynes** [comment]
This is great, thank you for the contribution @roshaans!

Since the change being made here is already in place in `components/nft/NFTBox.js` it would be best to refactor this so that both components are rendering NFT media using the exact same logic. If you have time would you mind please adding a new functional component in `components/nft/NFTMedia.js` that takes a `mediaUrl` prop and renders either the `<video />` or `` tag using the same logic in your change and `NFTBox.js`? Then both `NFTBox.js` and `NFTDetail.js`  can render `<NFTMedia mediaUrl={...} />` instead of having it inline.

This will ensure that any future changes to how we render NFT media are always applied to both components. If you don't have time I am happy to make the change as well, but I may need push access to your fork. Let me know if this works for you otherwise I'm sure we can work something out to get your fix into the codebase.

**Comment by agileurbanite** [comment]
@andy-haynes let's wait till 4/4/22 for @roshaans to reply before we take over and put this in the proper place.

**Comment by agileurbanite** [comment]
Related to #2559

**Comment by roshaans** [comment]
> 

Yes, I will get this in as soon I can!

**Comment by roshaans** [comment]
@andy-haynes @agileurbanite I believe that takes care of the refactor. Let me know if you want anything else changed and happy to contribute!

**Review by andy-haynes — APPROVED** [review]
Great work, @roshaans! Thank you for your bug fix and refactor ❤️

**Pull request merged**
